tests/e2e/env.go

```diff
@@ -19,16 +30,20 @@ func NewChainlinkTerraEnv() *environment.Config {
 			"chainlink": {
 				Index: 4,
 				Values: map[string]interface{}{
-					"replicas": 5,
+					"replicas": nodes,
 					"chainlink": map[string]interface{}{
 						"image": map[string]interface{}{
 							"image":   "public.ecr.aws/z0b1w9r9/chainlink",
-							"version": "develop.3abacbfc0761b4f6cf4d4d897bb4f94d05a4f793",
+							"version": "candidate-develop-terr-qa-test-1.b8f1be3c51ec4dd0cc693b3c754c4ce2cd9af05d",
 						},
 					},
+					"db": db,
 					"env": map[string]interface{}{
+						"EVM_ENABLED":                 "false",
+						"EVM_RPC_ENABLED":             "false",
+						"TERRA_ENABLED":               "true",
 						"eth_disabled":                "true",
-						"CHAINLINK_DEV":               "true",
+						"CHAINLINK_DEV":               "false",
```

Comment:
This seems useful in e2e tests, why remove it?

Reply:
If I set it to "true" where we test it when it's maybe set to "false" in prod?